config/tracks/simtraxx_pikes_peak_hc_v0.3.ini

```diff
@@ -0,0 +1,253 @@
+DATE_RELEASE = 11/03/2021
+NOTES = rain, lights, grass and seasons for Pikes Peak. Todo:fix normals of trees:must be done in the track 3d model.
+
+[SHADER_REPLACEMENT_0]
+MATERIALS=PIK_PK?
+SHADER=ksPerPixel
+
+[INCLUDE]
+INCLUDE = common/all_conditions.ini
+
+[SOL]
+SMOG_MORNING=0.2
+SMOG_NOON=0.1
+SMOG_EVENING=0.2
+
+
+[LIGHTING]
+SPECULAR_MULT = 1
+LIT_MULT = 1
+
+[DISTANT_GLOW]
+THRESHOLD = 25
+SHARED_THRESHOLD = 25
+
+[DEFAULTS]
+GrassMaterials=grass?,PIK_T4.dds,PIK_P4.dds,PIK_T62.dds,PIK_T1.dds,PIK_PK?,PIK_PL?,pk3415,pk3417,pk3414,PIK_T9.dds,PIK_P6.dds, PIK_P1.dds, PIK_P3.dds,PIK_P2.dds,PIK_P5.dds,PIK_T61.dds
+TreesMaterials=WallPine?,TreeP_Multi_?.dd,Pine.dds,Horizont_?.dds,birch_?.dds,Birch_??,Material #24682,pk3411,pk3412,pk3413,WallBirch_?,TreeP_?,Pine_?,pk3416
+BushesMaterials=bush?
+2dSpectatorsMeshes=2D spec?
+3dSpectatorsMeshes=3D spec?,Girlsunglass.dds,
+RoadMaterials=PIK_Ta1.dds,PIK_Conc.dds,PIK_ASF.dds
+RoadPaintMaterials=?paint?,?curb?, ?line?
+MetalMaterials=Car?,Amb_?,Guarda.dds,rescuetex.dds,TOILET.dds,FireTruck.dds,House.dds,Sign2.dds,PIK_Ru.dds,sign1.dds,Signs_Guarda.dds,Cars_?, Truck_?,PoliceCar.dds
+PlasticMaterials=Logos?,ConeRed,startline.dds
+FabricMaterials=?flags?
+TentsMaterials=Camp_?,Tent_?,Umburella.dds
+GlassMaterials=
+RoughWoodMaterials=?wood?
+RoughRockMaterials=pk3414,pk3415,pk3417,PIK_SK1.dds,Rock_1.dds,PIK_Ka.dds,PIK_T52.dds,SAT_9.dds
+SandMaterials=PIK_T8.dds
+ConcreteBuildingMaterials=?building?,?house?
+WoodBuildingMaterials=inn_02_d.dds,inn_01_d.dds,PIK_CAS.DDS
+
+[SHADER_REPLACEMENT_...]
+DESCRIPTION=remove ksEmissive from objects so that it stops glowing at night
+MATERIALS=$RoadMaterials, $GrassMaterials, $RoadPaintMaterials, $SandMaterials, $TreesMaterials, $RoughRockMaterials
+PROP_... = ksEmissive, 0, 0, 0
+ACTIVE = 1
+
+[SHADER_REPLACEMENT_...]
+DESCRIPTION=fix ambiant light multiplier for tarmac road
+MATERIALS=$RoadMaterials,PIK_01_TB1_SS.dd
+PROP_... = ksAmbient, 0.35
+PROP_... = ksDiffuse, 0.21
+
+ACTIVE = 1
+
+[SHADER_REPLACEMENT_...]
+DESCRIPTION=fix ambiant light multiplier for some objects
+MATERIALS=PIK_PK2.dds,Material #24682,pk3412
+PROP_... = ksAmbient, 0.35
+ACTIVE = 1
+
+[SHADER_REPLACEMENT_...]
+DESCRIPTION=fix ambiant light multiplier for some objects
+MATERIALS=PIK_P4.dds
+PROP_... = ksAmbient, 0.25
+ACTIVE = 1
+
+
+[SHADER_REPLACEMENT_...]
+DESCRIPTION=fix ambiant light multiplier for some objects
+MATERIALS=PIK_PK12.dds,PIK_PK12_1.dds
+PROP_... = ksAmbient, 0.67
+ACTIVE = 1
+
+[SHADER_REPLACEMENT_...]
+DESCRIPTION=fix ambiant light multiplier for some objects
+MATERIALS=PIK_PK4_1.dds,PIK_PK4.dds
+PROP_... = ksAmbient, 0.32
+ACTIVE = 1
+BLEND_MODE = TRANSPARENT_AS_BLACK
+
+
+[LIGHT_POLLUTION]
+RELATIVE_POSITION = 0, 0, 0
+DENSITY = 0.2
+RADIUS_KM = 0.7
+COLOR = 1.0, 0.55, 0.16
+ACTIVE = 1
+
+[BOUNCED_LIGHT]
+SUPPORTED=1
+SURFACE_MATERIALS=$RoadMaterials, $GrassMaterials, $RoadPaintMaterials, $RoughWoodMaterials, $SandMaterials, $MetalMaterials, $RoughWoodMaterials
```

Comment:
Same as previous comment.

Reply:
I didn't understand well how it works. These 2 tracks don't have any tunnels or bridges. So, are bounced lights useful in this case ? I'm going to remove it.